docs/common/istio-ambient-update.adoc

```diff
@@ -0,0 +1,653 @@
+
+* **Rolling Updates:** ZTunnel uses a RollingUpdate strategy, updating one node at a time by default.
+* **Minimal Disruption:** While a node's ZTunnel pod is restarting, new connections may experience brief latency, but existing connections are maintained.
+* **Node-by-Node:** The update process ensures that at least one ZTunnel pod is always available on each node before proceeding to the next.
+* **Monitoring:** Monitor the ZTunnel DaemonSet rollout status:
+
+[source,bash,subs="attributes+"]
+----
+$ kubectl rollout status daemonset/ztunnel -n ztunnel
+----
+
+[[control-plane-and-data-plane-version-skew]]
+=== Control Plane and Data Plane Version Skew
+
+In ambient mode, version skew between components is handled differently than in sidecar mode:
+
+* **Supported Skew:** The ztunnel at version 1.x is compatible with the control plane at version 1.x+1 and 1.x.
+* **Testing Required:** Always test your specific version combinations in a non-production environment.
+* **Recommendation:** Keep all components (Istio, IstioCNI, ZTunnel) at the same version when possible.
+
+[[waypoint-proxy-compatibility]]
+=== Waypoint Proxy Compatibility
+
+If you have deployed waypoint proxies for L7 features:
+
+* **Automatic Updates:** Waypoint proxies automatically reference the active control plane revision.
+* **Gateway API:** Waypoint proxies are deployed using Kubernetes Gateway resources and are managed automatically by Istiod.
+* **Verification:** Test L7 features after the upgrade to ensure waypoint proxies are functioning correctly. See link:./istio-ambient-waypoint.adoc#layer-7-features-in-ambient-mode[Layer 7 Features in Ambient Mode] for testing examples.
+
+**Update Behavior by Strategy:**
+
+* **InPlace Strategy:** Waypoint proxies transition directly to the new control plane version
+* **RevisionBased Strategy:** Waypoint proxies can function with both control plane revisions during migration. However, note that the single-ztunnel limitation still applies to the ambient data plane.
+
+**Cross-Namespace Waypoints:**
+
+Verify labels remain in place for cross-namespace waypoint usage:
+
+[source,bash,subs="attributes+"]
+----
+$ kubectl get ns bookinfo --show-labels | grep waypoint
+bookinfo  Active  istio.io/use-waypoint-namespace=foo,istio.io/use-waypoint=waypoint-foo
+----
+
+For detailed waypoint update procedures, see link:./istio-ambient-waypoint.adoc#updating-waypoint-proxies[Updating Waypoint Proxies].
+
+[[impact-on-existing-ambient-workloads]]
+=== Impact on Existing Ambient Workloads
+
+During ambient mode upgrades:
+
+When upgrading the ambient cluster, new mTLS connections continue to function normally throughout the upgrade process. However, upgrading ztunnel will cause any existing long-lived TCP connections (including mTLS connections) on the upgraded node to reset after a grace period. To minimize disruption during production upgrades, node cordoning, blue/green node pools are recommended.
```

Comment:
As discussed on slack, lets remove the following statement until we find a proper solution.

```suggestion
When upgrading the ambient cluster, new mTLS connections continue to function normally throughout the upgrade process. However, upgrading ztunnel will cause any existing long-lived TCP connections (including mTLS connections) on the upgraded node to reset after a grace period. 
```